src/components/LeafletMap.js

```diff
@@ -332,6 +342,48 @@ class LeafletMap extends Component {
     return selectMarkerCluster;
   }
 
+  createPopup(markersData, coordinates) {
+    const popUpContentStyles =
+      markersData.length > 2
+        ? leafletProperty.popUpHigherStyles
+        : leafletProperty.popUpDefaultStyles;
+    const popUpArray = markersData.map((item, index) => {
+      const itemStyle =
+        index === markersData.length - 1
+          ? leafletProperty.popUpContentLastItemStyles
+          : leafletProperty.popUpContentItemStyles;
+
+      return `
+            <div style="${itemStyle}" id="${item.id}" class="popup-item active">
+              ${item.properties.site_name} - ${item.properties.project_name}
+            </div>`;
+    });
+
+    const popUpContent = `
+            <div style="${popUpContentStyles}" class="popup-wrapper">
+              ${popUpArray.join('')}
+            </div>
+        `;
+
+    const popup = L.popup()
+      .setLatLng(coordinates)
+      .setContent(`${popUpContent}`);
```

Comment:
Do you need string interpolation here? Can't you just jam in the string as a variable?